sandbox/stylesheets/default.qss

```diff
@@ -42,6 +42,7 @@
 #define ACTIVE_BACKGROUND_COLOR             rgb(44, 44, 44)
 #define HOVER_BACKGROUND_COLOR              rgb(190, 140, 50)
 #define SELECTED_BACKGROUND_COLOR           rgb(60, 60, 60)
+#define EDITOR_BACKGROUND_COLOR             rgb(48, 48, 48)
```

Comment:
I would suggest hardcoding this value since you appear to use it only once. Defines are for styles common to many widgets.

I can change it, no worries.